packages/airnode-node/src/adapters/http/worker.test.ts

```diff
@@ -1,14 +1,12 @@
-const invokeMock = jest.fn();
-jest.mock('aws-sdk', () => ({
-  Lambda: jest.fn().mockImplementation(() => ({
-    invoke: invokeMock,
-  })),
-}));
-
+import { mockClient } from 'aws-sdk-client-mock';
+import 'aws-sdk-client-mock-jest';
+import { LambdaClient, InvokeCommand } from '@aws-sdk/client-lambda';
 import { logger, LogOptions, setLogOptions } from '@api3/airnode-utilities';
 import * as worker from './worker';
 import * as fixtures from '../../../test/fixtures';
 
+const mockLambdaClient = mockClient(LambdaClient);
```

Comment:
So this basically does the mocking for you internally, and returns you a spy that you can mock... Right?

Reply:
Yes.